YoCode/ProjectBuilder.cs

```diff
@@ -63,11 +66,15 @@ public static string GetErrorOutput(string output)
             return "";
         }
 
-        private bool BuildSuccessful()
+        private void CheckBuildSuccess()
         {
-            var buildLine = Output.GetLineWithOneKeyword("Build succeeded");
-
-            return buildLine != "";
+            if(Output.Contains("is being used by another process"))
+            {
+                ProjectBuilderEvidence.SetInconclusive(messages.BadPort);
```

Comment:
Why is there a "bad port" message in here? Ports are not related to builds so I'm not sure why this would ever emit a message about a bad port.